test/Logger.test.js

```diff
@@ -1 +1,93 @@
+import sinon from 'sinon';
+import {expect} from 'chai';
 import {Logger} from '../src/Logger.js';
+
+const DEFAULT_LEVEL = 30;
+
+describe('Logger', () => {
+  it('should construct with no args', () => {
+    expect(() => {
+      return new Logger();
+    }).to.not.throw();
+  });
+
+  describe('Logger Instance', () => {
+    let logger;
+    beforeEach(() => {
+      logger = new Logger();
+    });
+
+    it('should return the current log level from the "level" method', () => {
+      expect(logger.level()).to.equal(DEFAULT_LEVEL);
+    });
+
+    it('should update the log level if passed a number', () => {
+      logger.level(100);
+      expect(logger.level()).to.equal(100);
+      logger.level(15);
+      expect(logger.level()).to.equal(15);
+    });
+
+    it('should update the log level if a valid string is passed', () => {
+      logger.level('warn');
+      expect(logger.level()).to.equal(40);
+      logger.level('error');
+      expect(logger.level()).to.equal(50);
+    });
+
+    it('should use the default log level is an invalid string is passed', () => {
+      logger.level('invalid');
+      expect(logger.level()).to.equal(DEFAULT_LEVEL);
+    });
+
+    it('should only log a message if the log level is higher than the log', () => {
```

Comment:
It should really be if the log level is lower or equal to. So, if the log level is 20 you get debug + info + error, etc.